lightning/restore/restore.go

```diff
@@ -1239,7 +1235,8 @@ func (tr *TableRestore) importKV(ctx context.Context, closedEngine *kv.ClosedEng
 	metric.ImportSecondsHistogram.Observe(dur.Seconds())
 	common.AppLogger.Infof("[%s] kv deliver all flushed, takes %v", closedEngine.Tag(), dur)
 
-	// gofail: var SlowDownImport struct{}
+	// TODO: implement failpoint inject empty
+	failpoint.Inject("SlowDownImport", func() {})
```

Comment:
For now let's make it 

```go
failpoint.Inject("SlowDownImport", func(val failpoint.Value) {
    time.Sleep(val.(int) * time.Millisecond)
})
```

to keep it working. (If we could implement the `sleep` thing before this PR is merged then disregard this comment.)

Reply:
I will fix it to tomorrow